src/Catalog/Helpers/GalleryDatabaseQueryService.cs

```diff
@@ -20,19 +20,6 @@ public class GalleryDatabaseQueryService : IGalleryDatabaseQueryService
         private const string PackageIdParameterName = "PackageId";
         private const string PackageVersionParameterName = "PackageVersion";
 
-        private static readonly string Db2CatalogSqlSubQuery = $@"PR.[Id],	
```

Comment:
I expect that as part of the `MonitoringProcessor` changes for deprecation, we will need to revert this refactoring so that `GetPackageOrNull` can return deprecation information. I would rather that we don't refactor this and then just verify that `MonitoringProcessor` doesn't validate the deprecation information with this change.

Reply:
Reverted query duplication in https://github.com/NuGet/NuGet.Services.Metadata/pull/542/commits/e57f596536a1840dfa2a5191c85946f149b99062